### PR TITLE
34 about ussectionsrules

### PR DESCRIPTION
**Pull request by Klumpish**
# Description:
I have - added a rules section with rules and image on about-us.html

Issue #34   

# Type of change:

- [x] hard code text and image for show
- [x] add an image
- [x] add text
- [x] style the text and image so that they are next to each other

# Additional information (screenshot, video, code examples):

image

### CODE REVIEW

**Review by alfredbrannare — APPROVED** [review]
Looks good!

**Review by jennymakki — APPROVED** [review]
Looks very good! I like the picture.

File: about-us.html
Comment (Paajt) [inline about-us.html:1]: BEM ❤️🤣

File: src/assets/images/rulesImage.jpg
Comment (Paajt) [inline src/assets/images/rulesImage.jpg:1]: Intimidating, in a good way!

**Review by Paajt — APPROVED** [review]
LG2M! 👍

**Pull request merged**